src/api/app/queries/event_subscriptions_finder.rb

```diff
@@ -1,17 +0,0 @@
-class EventSubscriptionsFinder
-  def initialize(relation = EventSubscription.all, event_package_or_request:)
-    @relation = if event_package_or_request.is_a?(Package)
-                  relation.where(package: event_package_or_request)
-                else
-                  relation.where(bs_request: event_package_or_request)
-                end
-  end
-
-  def for_scm_channel_with_token(event_type:)
-    @relation
-      .joins(:token)
```

Comment:
@hennevogel if I see the diff correctly, this `joins` part is dropped. but to me it looks like it would makes sense to keep it. Maybe I'm overlooking something

Reply:
Why?

Reply:
Sorry, I add another comment, but didn't got submitted. Because we only want to report for enabled tokens, this got introduced here https://github.com/openSUSE/open-build-service/pull/17394